django/contrib/gis/db/backends/mysql/operations.py

```diff
@@ -105,3 +109,16 @@ def get_geom_placeholder(self, f, value, compiler):
         else:
             placeholder = '%s(%%s)' % self.from_text
         return placeholder
+
+    def get_db_converters(self, expression):
+        converters = super(MySQLOperations, self).get_db_converters(expression)
+        if isinstance(expression.output_field, GeometryField) and self.uses_invalid_empty_geometry_collection:
+            converters.append(self.convert_invalid_empty_geometry_collection)
+        return converters
+
+    # https://dev.mysql.com/doc/refman/5.7/en/spatial-function-argument-handling.html
```

Comment:
Could you make this a docstring (wrapping the lines at 79 characters) and put the link under the description?